.ci/end2end.groovy

```diff
@@ -110,6 +110,9 @@ pipeline {
         archiveArtifacts(allowEmptyArchive: true, artifacts: "${E2E_DIR}/kibana.log")
       }
     }
+    cleanup {
+      notifyBuildResult()
```

Comment:
I'd say to disable all the notifications for GH PR comments and email notifications (if possible, I don't recall if that's something we can do), otherwise we might add some misleading since the kibana operations team already use some other notifications in place

Reply:
@v1v Good call. Something like this?